R/psis.R

```diff
@@ -330,6 +349,48 @@ do_psis_i <- function(log_ratios_i, tail_len_i) {
   list(log_weights = lw_i, pareto_k = khat)
 }
 
+#' TIS on a single vector
+#'
+#' @noRd
+#' @param log_ratios_i A vector of log importance ratios (for `loo()`, negative
+#'   log likelihoods).
+#' @param tail_len_i Not used. Included to conform to PSIS API.
+#'
+#' @details Implementation of Truncated importance sampling (PSIS), a method for
+#' stabilizing importance ratios. The version of TIS implemented here
+#' corresponds to the algorithm presented in Ionides (2008) with truncation at
+#' sqrt(S).
+#'
+#' @return A named list containing:
+#' * `lw`: vector of unnormalized log weights
```

Comment:
it might be useful to state that unnormalized and sifted to have max=0, or change the function to not to do sifting.

Reply:
see #112

Reply:
Indeed. I did it for it to be similar to the PSIS implementation. I update the doc and fix issue #112.